packages/terra-date-picker/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Changelog
 
 Unreleased
 ----------
+### Changed
```

Comment:
Should this be `###Fixed`?

Reply:
Maybe. I was thinking that we changed the behavior more so than fixing the underlying problem.
https://github.com/cerner/terra-framework/pull/1132/commits/f37f71c9d677bf16a39b3ee2fb62494cbd0c6c25